Capstone/frontend-employee/src/Pages/AdminDashboard/Manager/DisplayManager.jsx

```diff
@@ -0,0 +1,44 @@
+import React, { useEffect, useState } from "react";
+import axios from "axios";
+import '../../../style/DIsplay.css';
+import ManagerCard from "../../../component/SingleManagerCard/ManagerCard";
+
+const DisplayManager = () => {
+  const [employees, setEmployees] = useState([]);
+  const [projects, setProjects] = useState([]);
+  const [managerProjects, setManagerProjects] = useState({});
+  const [selectedSkills, setSelectedSkills] = useState({});
+
+  useEffect(() => {
+    getAllManagers();
+    // getAllProjects();
+  }, []);
+
+
+
+  const getAllManagers = async () => {
+    try {
+      const response = await axios.get(
+        "http://localhost:8080/user/all/MANAGER"
```

Comment:
Use common functions for call actions for GET, POST, PUT methods. And don't use endpoints directly like this.